client/tests/sdk/test_dataset_sdk.py

```diff
@@ -253,16 +254,32 @@ def test_setitem_with_auto_encode(self) -> None:
             "large_str": "abc" * 1000,
             "bytes": b"abc",
             "large_bytes": b"abc" * 1000,
+            "one_type_list": [1, 2, 3],
+            "one_type_tuple": ["a", "b", "c"],
+            "mixed_types_list": [1, "2", 3.0, ["a", "b"]],
```

Comment:
the case `"mixed_types_list": [1, "2", 3.0, [1, "b"]]` would fail

Reply:
fixed